src/errors/InvalidResourceError.ts

```diff
@@ -5,10 +5,19 @@
 
 // eslint-disable-next-line import/prefer-default-export
 export class InvalidResourceError extends Error {
+    readonly isInvalidResourceError: boolean;
+
     constructor(message = 'Invalid Resource') {
         // Node Error class requires passing a string message to the parent class
         super(message);
         Object.setPrototypeOf(this, InvalidResourceError.prototype);
+        this.isInvalidResourceError = true;
         this.name = this.constructor.name;
     }
 }
+export function isInvalidResourceError(err: InvalidResourceError | Error): err is InvalidResourceError {
```

Comment:
I think that err parameter should ideally be of type `unknown`, or maybe `any`. 

The Express error handler actually doesn't know if an error is of type `Error` beforehand(by convention it is, but really you can throw anything).

Reply:
Good to know